
## Controller Configs

k1: -237.00
k2: -72.4115
k3: -22.8654
k4: -26.1364
k5: 10.0

max_F: 100.0  # (N)
x3d: 4.0  # (m)

## System Configs

# Initial Conditions

theta_0: 0.0  # (rad)
x_0: 4.0  # (m)

## System Definition

l: 1.0  # (m)
m: 1.0  # (kg)
M: 5.0  # (kg)
max_disturbance: 0.5  # (rad/sec^2)

## simulation_configs
ctrl_rate: 60.0 # (Hz)
sim_rate: 100.0 # (Hz)
manager_rate: 15.0 # (Hz)

rand_seed: 101

neighbour_1: "p4"
neighbour_2: "VOID"
distance_threshold: 0.95